react-bootstrap/react-bootstrap.d.ts

```diff
@@ -704,6 +704,88 @@ declare module "react-bootstrap" {
     }
     var FormControls: FormControlsClass;
 
+    // <Form />
+    // ----------------------------------------
+    interface FormProps extends React.HTMLAttributes {
+        bsClass?: string;
+        componentClass?: string;
+        horizontal?: boolean;
+        inline?: boolean;
+    }
+    type Form = React.ClassicComponent<FormProps, {}>;
+    var Form: React.ClassicComponentClass<FormProps>;
+
+    // <FormGroup />
+    // ----------------------------------------
+    interface FormGroupProps extends React.HTMLAttributes {
+        bsClass?: string;
+        bsSize?: 'sm' | 'small' | 'lg' | 'large';
+        controlId?: string;
+        validationState?: 'success' | 'warning' | 'error';
+    }
+    type FormGroup = React.ClassicComponent<FormGroupProps, {}>;
+    var FormGroup: React.ClassicComponentClass<FormGroupProps>;
```

Comment:
ditto, should be `class FormGroup extends React.Component<FormGroupProps, {}> {`